<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Key_Design</class>
 <widget class="QWidget" name="Key_Design">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="delete_Button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>XÓA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hook_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>HOOK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="printkey_Button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>IN PHÍM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="unhook_Button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>UNHOOK</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>341</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
